### DISCUSSION TITLE
zopen/var/log - ever cleaned up? viewer?

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#679] [General]
will the zopen/var/log ever be cleaned up or will it just continue forever?

any thought to a viewer for it?

**Comment by DevonianTeuchter** [comment]
It is never cleaned up currently, nor a mechanism to do so -  could be an option to zopen-clean to archive it (rather than delete) since there is an audit log in there which might be best kept.  The format is fairly rigid so it wouldn't be beyond the realms of possibility to generate a viewer if there was felt to be a need for it.
